### PR TITLE
Added ability to render from input.js

### PR DESCRIPTION
**Pull request by thegreyspot**
Removed <script> tag rending of codemirror on the field. CodeMirror is
now rendered from input.js and allows for proper
functionality with repeater/flexible content fields.

### CODE REVIEW

File: js/input.js [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
+				lineNumbers: true,
+				fixedGutter: false,
+				mode       : $textarea.attr("mode"),
+				theme      : $textarea.attr("theme"),
+				extraKeys  : {"Ctrl-Space": "autocomplete"},
+				value      : document.documentElement.innerHTML
+			});
+			editor.refresh();
+
+			console.log("Added new code field")
```
Comment (ptasker) [inline js/input.js]: Let's remove the console.log() for our legacy browser friends.

File: js/input.js [hunk 2]
```diff
@@ -1,15 +1,30 @@
...
+			//if just a clone element for repeater field
+			if($el.parents(".acf-clone").length >0)
+				return;
+
+			var $textarea = $el.find('.acf-input>textarea');
+
+			//load needed CSS if doesn't already exist
+			if (!$("link[href='/wp-content/plugins/acf-code-field/js/codemirror-5.13/theme/"+  $textarea.attr("theme") +".css']").length)
+				$('<link href="/wp-content/plugins/acf-code-field/js/codemirror-5.13/theme/'+  $textarea.attr("theme") + '.css" rel="stylesheet">').appendTo("head");
+
```
Comment (ptasker) [inline js/input.js]: I'm a little weary hard coding the plugins URL in here. Maybe use wp_localize_script and pass in the plugins_url() URL for the plugin.

**Comment by ptasker** [comment]
Cool, just a couple comments on the PR. Once those are resolved I'll merge it in.

**Comment by ptasker** [comment]
Manually merged some of this in.

**Comment by thegreyspot** [comment]
Sorry, missed the comments. Thanks for merging.

**Pull request closed without merging**